File: z-quiz.ngApp/src/app/models/question.ts
import { BaseModel } from './base-model'; 

export class Question extends BaseModel  {
    public QuestionId: number;
    public Title: string;
    public TotalScore: number;
    public Choices: Choice[];
}

export class Choice extends BaseModel {
    public ChoiceId: number;
    public Title: string;
    public QuestionId: number;
}

export const MockQuestions = () => {
    let questions: Question[] = [];
    let chId: number = 0;

    for (let i = 0; i < 5; i++) {
        let q = new Question();
        q.QuestionId = i + 1;
        q.Title = 'Question of ' + q.QuestionId;
        q.Choices = [];
        for (let j = 0; j < 5; j++, chId++) {
            let ch = new Choice();
            ch.ChoiceId = chId;
            ch.Title = 'Answer ' + chId + ' of question ' + q.QuestionId;
            q.Choices.push(ch);
        }
        questions.push(q);
    }

    return questions;
}

File: z-quiz.api/Models/Question.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Data.Entity.Spatial;

namespace z_quiz.api.Models
{
    public partial class Question
    {
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
        public Question()
        {
            Choices = new HashSet<Choice>();
            TesterQuestions = new HashSet<TesterQuestion>();
        }

        public int QuestionId { get; set; }

        [Required]
        [StringLength(2000)]
        public string Title { get; set; }

        public int TotalScore { get; set; }

        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<Choice> Choices { get; set; }

        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<TesterQuestion> TesterQuestions { get; set; }
    }
}